Source/WebCore/crypto/gcrypt/CryptoAlgorithmRegistryGCrypt.cpp

```diff
@@ -57,6 +58,7 @@ void CryptoAlgorithmRegistry::platformRegisterAlgorithms()
     registerAlgorithm<CryptoAlgorithmAES_KW>();
     registerAlgorithm<CryptoAlgorithmECDH>();
     registerAlgorithm<CryptoAlgorithmECDSA>();
+    registerAlgorithm<CryptoAlgorithmEd25519>();
```

Comment:
Given there is no platform support, it might be better to not register this algorithm